File: my-app/fix_conflicts.js
const fs = require('fs');
const path = require('path');

function removeCommentedCode(filepath) {
    const content = fs.readFileSync(filepath, 'utf8');
    const lines = content.split('\n');

    // Find the first non-commented import statement
    let startIndex = 0;
    for (let i = 0; i < lines.length; i++) {
        const trimmed = lines[i].trim();
        if (trimmed.startsWith('import ') && !trimmed.startsWith('// import')) {
            startIndex = i;
            break;
        }
    }

    if (startIndex > 0) {
        const cleaned = lines.slice(startIndex).join('\n');
        fs.writeFileSync(filepath, cleaned, 'utf8');
        console.log(`Fixed ${path.basename(filepath)} - removed ${startIndex} lines of commented code`);
    } else {
        console.log(`${path.basename(filepath)} - no changes needed`);
    }
}

const files = [
    path.join(__dirname, 'src', 'Pages', 'MiniDrawer.js'),
    path.join(__dirname, 'src', 'Pages', 'StoreDetails.js'),
    path.join(__dirname, 'src', 'Pages', 'Storemanagement.js')
];

files.forEach(filepath => {
    try {
        if (fs.existsSync(filepath)) {
            removeCommentedCode(filepath);
        } else {
            console.log(`File not found: ${filepath}`);
        }
    } catch (error) {
        console.error(`Error processing ${filepath}:`, error.message);
    }
});

console.log('Done!');

File: my-app/fix_conflicts.py
import re

def remove_git_conflicts(filepath):
    """Remove git conflict markers and keep the appropriate version"""
    with open(filepath, 'r', encoding='utf-8') as f:
        content = f.read()
    
    # Pattern to match conflict blocks
    pattern = r'<<<<<<< HEAD\n(.*?)\n=======\n(.*?)\n>>>>>>> [a-f0-9]+\n'
    
    # For most conflicts, we'll keep the HEAD version (first part)
    # But we need to handle each file differently
    
    if 'StoreCustomer.js' in filepath:
        # For StoreCustomer, keep the required fields (HEAD version)
        cleaned = re.sub(pattern, r'\1\n', content, flags=re.DOTALL)
    elif 'MiniDrawer.js' in filepath:
        # Remove all commented code at the top
        lines = content.split('\n')
        active_start = None
        for i, line in enumerate(lines):
            if line.strip() and not line.strip().startswith('//'):
                active_start = i
                break
        if active_start:
            content = '\n'.join(lines[active_start:])
        cleaned = re.sub(pattern, r'\1\n', content, flags=re.DOTALL)
    elif 'StoreDetails.js' in filepath:
        # Remove duplicate component definition
        lines = content.split('\n')
        # Find where actual imports start (not commented)
        active_start = None
        for i, line in enumerate(lines):
            if line.strip().startswith('import ') and not line.strip().startswith('//'):
                active_start = i
                break
        if active_start:
            content = '\n'.join(lines[active_start:])
        cleaned = re.sub(pattern, r'\1\n', content, flags=re.DOTALL)
    elif 'Storemanagement.js' in filepath:
        # Remove duplicate component definition
        lines = content.split('\n')
        # Find where actual imports start (not commented)
        active_start = None
        for i, line in enumerate(lines):
            if line.strip().startswith('import ') and not line.strip().startswith('//'):
                active_start = i
                break
        if active_start:
            content = '\n'.join(lines[active_start:])
        cleaned = re.sub(pattern, r'\1\n', content, flags=re.DOTALL)
    else:
        cleaned = re.sub(pattern, r'\1\n', content, flags=re.DOTALL)
    
    with open(filepath, 'w', encoding='utf-8') as f:
        f.write(cleaned)
    
    print(f"Fixed {filepath}")

# Fix the files
files = [
    r'e:\prd\Latest_Update\StockHandle-main\StockHandle-main\my-app\src\Pages\StoreCustomer.js',
    r'e:\prd\Latest_Update\StockHandle-main\StockHandle-main\my-app\src\Pages\MiniDrawer.js',
    r'e:\prd\Latest_Update\StockHandle-main\StockHandle-main\my-app\src\Pages\StoreDetails.js',
    r'e:\prd\Latest_Update\StockHandle-main\StockHandle-main\my-app\src\Pages\Storemanagement.js'
]

for filepath in files:
    try:
        remove_git_conflicts(filepath)
    except Exception as e:
        print(f"Error fixing {filepath}: {e}")
